templates/focus-ios.xliff

```diff
@@ -269,13 +396,19 @@
         <source>Settings</source>
         <note>Title for settings screen</note>
       </trans-unit>
+      <trans-unit id="Settings.searchLabel">
+        <source>Search Engine</source>
+        <note>Label for the search engine in the search screen</note>
+      </trans-unit>
       <trans-unit id="Settings.searchSection">
         <source>SEARCH ENGINE</source>
-        <note>Title for the search engine row</note>
       </trans-unit>
       <trans-unit id="Settings.searchTitle">
         <source>Search Engine</source>
-        <note>Title for the search engine selection screen</note>
```

Comment:
Same as above: wondering why we're dropping the localization note in this case?

Reply:
We removed this string from the app. Replacing it with `Settings.searchTitle2`

Reply:
Ehm… https://github.com/mozilla-mobile/focus-ios/issues/395#issuecomment-326502215

Reply:
@flodolo good to know. I can remove them and do another export if you'd like? cc: @Delphine

Reply:
Likely good enough to remove them now in code, so they're fixed with the next export and not forgotten.